### PR TITLE
Write the UI instructions in the README

### PR DESCRIPTION
**Pull request by maelvls**
In #9, we expressed the need to have a URL that we could use in the solution as the "User-guide documentation".

Related:
- https://github.com/jetstack/preflight-platform/issues/1141
- https://github.com/jetstack/preflight-platform/issues/1023

| To preview the README.md: |
|---|
| https://github.com/jetstack/jetstack-secure-gcm/blob/add-ui-instructions/README.md |

Up to now, the README only contained CLI instructions. This PR adds the "UI instructions". A link to this part of the README will be shown as the "Getting started documentation"; in the [draft](https://console.cloud.google.com/marketplace/details/jetstack-public/jetstack-secure-for-cert-manager__draft), it looks like:

solution-page

cc @wallrj

### CODE REVIEW

**Comment by james-w** [comment]
I think this looks good.
/hold
/lgtm

There's some comments from Richard, not sure if you want to address those?

File: README.md [hunk 1]
```diff
@@ -59,10 +63,286 @@ more effective overall management of clusters.
...
+<img src="https://user-images.githubusercontent.com/2195781/109155511-ec08b180-776f-11eb-82e1-45b0c32720d6.png" width="500" alt="Choose your cluster name and click the check mark button. This screenshot is stored in this issue: https://github.com/jetstack/jetstack-secure-gcm/issues/21">
+
+Next, click the button "Copy command to clipboard":
+
+<img src="https://user-images.githubusercontent.com/2195781/109026248-d76ce080-76bf-11eb-94be-bc1c8f54b2cf.png" width="500" alt="Click the button 'Copy command to clipboard'. This screenshot is stored in this issue: https://github.com/jetstack/jetstack-secure-gcm/issues/21">
+
+With the help of a text editor, paste the content and edit it:
+
+1. remove the 6 first lines,
+2. remove the last line.
```
[the remark this answers is not in the document]
Comment (maelvls) [reply README.md:140]: @charlieegan3 I tried using the [proposed approach](https://github.com/jetstack/preflight-platform/issues/1141) but it was getting quite complicated to detail what needed to be changed in the configmap :(

I propose that we keep the ugly "delete those lines" for now and find a better way later?

File: README.md [hunk 2]
```diff
@@ -62,7 +65,85 @@ more effective overall management of clusters.
...
+
+Save the file as `agent-config.yaml` and apply it to your cluster:
+
+```sh
+# This is the namespace chosen before clicking on "Deploy".
+NAMESPACE=jetstack-secure
+
+kubectl -n $NAMESPACE apply  -f agent-config.yaml
+kubectl -n $NAMESPACE rollout restart deploy jetstack-secure-preflight
+```
```
[the remark this answers is not in the document]
Comment (maelvls) [reply README.md]: I'll click "Resolve" for now, feel free to unresolved if you think we should discuss that more.

File: README.md [hunk 3]
```diff
@@ -62,7 +65,85 @@ more effective overall management of clusters.
...
+<img src="https://user-images.githubusercontent.com/2195781/109025110-ba83dd80-76be-11eb-9815-c91408c0096a.png" width="500" alt="Create cluster button. This screenshot is stored in this issue: https://github.com/jetstack/jetstack-secure-gcm/issues/21">
+
+Choose a name for your cluster; this name will show in the Jetstack Secure
+dashboard:
+
+<img src="https://user-images.githubusercontent.com/2195781/109025768-531a5d80-76bf-11eb-87c6-fa3cb888d9ea.png" width="500" alt="Choose your cluster name. This screenshot is stored in this issue: https://github.com/jetstack/jetstack-secure-gcm/issues/21">
+
+You will then see a button "Click to copy command to clopboard". Clicking
+it will copy the Kubernetes manifest for creating the configuration needed
+for the Jetstack Secure agent to report to the Jetstack Secure dashboard.
```
[the remark this answers is not in the document]
Comment (maelvls) [reply README.md]: Thanks!

File: README.md [hunk 4]
```diff
@@ -62,7 +65,85 @@ more effective overall management of clusters.
...
+### Step 1: Install Jestack Secure for cert-manager
+
+Head over to the [Jetstack Secure for
+cert-manager](https://console.cloud.google.com/marketplace/details/jetstack-public/jetstack-secure-for-cert-manager)
+solution page on the Google Cloud Marketplace. Click "Configure". You need
+to either select an existing cluster or create a new one that you want to
+deploy Jetstack Secure for cert-manager into. Finally, click the "Deploy"
+button:
+
+<img src="https://user-images.githubusercontent.com/2195781/109023553-31b87200-76bd-11eb-8fc4-a9e46ae44582.png" width="500" alt="this screenshot is stored in this issue: https://github.com/jetstack/jetstack-secure-gcm/issues/21">
```
[the remark this answers is not in the document]
Comment (maelvls) [reply README.md]: Good idea. I added a notice about using the proposed `jetstack-secure` namespace and about not changing the serviceaccount names.

File: README.md [hunk 5]
```diff
@@ -62,7 +65,85 @@ more effective overall management of clusters.
       - [(optional) Enable the Jetstack Secure web dashboard](#optional-enable-the-jetstack-secure-web-dashboard)
       - [(optional) Set up the Google Certificate Authority Service](#optional-set-up-the-google-certificate-authority-service)
 
-## Installation
+## Click-to-deploy installation
+
+This guide assumes you will follow the "click to deploy instructions"
+through the UI. If you prefer to deploy via the command line you can follow
+the instructions [below](#cli-installation).
```
[the remark this answers is not in the document]
Comment (maelvls) [reply README.md]: Your way is definitely better 😇

**Comment by maelvls** [comment]
> Comments from Richard, not sure if you want to address those?

I went over each comment thread, looks like all the changes @wallrj was expecting have been made

/unhold

**Pull request merged**